R/plot.R

```diff
@@ -0,0 +1,82 @@
+#' Plot method for RtGam fits
+#'
+#' Generates predicted draws from [predict.RtGam] and plots them with ggplot2.
+#' See
+#' the documentation on [predict.RtGam] for options on how to specify draws.
+#'
+#' @inheritParams predict.RtGam
+#' @param x An `RtGam` object created using the [RtGam()]
+#' function.
+#' @param alpha The opacity of the lines on the plot. Should be between 0 and 1,
+#' with values closer to 0 producing more transparent lines.
+#' @param ... Additional options to pass to [predict.RtGam]
+#' @returns A ggplot2 object
+#' @examples
+#' fit <- RtGam(
+#'   stochastic_sir_rt[["obs_incidence"]],
+#'   stochastic_sir_rt[["date"]]
+#' )
+#' # Plot draws from the fitted model against the data
+#' plot(fit)
+#'
+#' # Plot a 7-day forecast
+#' plot(fit, horizon = 7)
+#'
+#' # Plot Rt estimates
+#' plot(fit,
+#'   parameter = "Rt",
+#'   mean_delay = 0,
+#'   gi_pmf = sir_gt_pmf
+#' )
+#' @export
+plot.RtGam <- function(x, parameter = "obs_cases", alpha = 0.05, ...) {
```

Comment:
Looks good. I think we should have a issue for making these low level ggplot calls into their own standalone functions that take the output of predict as an alternative user path

Reply:
#83